packages/builder/vite.config.js

```diff
@@ -134,6 +134,10 @@ export default defineConfig(({ mode }) => {
           find: "@budibase/shared-core",
           replacement: path.resolve("../shared-core/src"),
         },
+        {
+          find: "@budibase/bbui",
+          replacement: path.resolve("../bbui/src"),
+        },
```

Comment:
How does this relate to the account portal?

Reply:
Again, not really related but I needed to tweak some dependencies. While testing things I realised some dependencies were conflicting or not updating properly. This will use the src folder instead of the dist one for dev proposes